app_server/app/adapters/social_media/SocialMediaAdapter.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Social Media Adapter
+ * @module app/adapters/social_media/SocialMediaAdapter
+ */
+
+var rfr = require('rfr');
+var util = require('util');
+
+var Utility = rfr('app/util/Utility');
+var Config = rfr('config/SocialMediaConfig');
+var Facebook = rfr('app/adapters/social_media/Facebook');
+
+var logger = Utility.createLogger(__filename);
+
+/**
+ * @constructor SocialMediaAdapter
+ * @param platform {string} can be: "facebook"
+ * @param options {object} options including accessToken
+ * @return {object}
+ */
+function SocialMediaAdapter(platform, options) {
+  if (platform === 'facebook') {
+    this.platform = new Facebook(options);
+  } else {
+    var errorMsg = util.format('Platform %s is not supported', platform);
+    logger.error(errorMsg);
+    throw new Error(errorMsg);
+  }
```

Comment:
```
facebook
```
could be declared as constants as Platform.js can include many other kinds of platform